dist/tools/insufficient_memory/create_makefile.ci.sh

```diff
@@ -56,6 +56,7 @@ for BOARD in $(EXTERNAL_BOARD_DIRS="" make  --no-print-directory info-boards-sup
         if grep -e overflowed \
                 -e "not within region" \
                 -e "wraps around address space" \
+                -e "overlaps section \.bss VMA" \
```

Comment:
```suggestion
                -e "overlaps section" \
```

could be any section, right?